README.md

```diff
@@ -335,6 +335,16 @@ private fun computeRange(name: String, from: Int, to: Int, op: String): FunSpec
 Literals are emitted directly to the output code with no escaping. Arguments for literals may be
 strings, primitives, and a few KotlinPoet types described below.
 
+### Nullability
```

Comment:
There's another PR that adds a chapter on nullability (#453), so this is redundant

Reply:
Didn't realize someone had already created a PR to resolve the issue I had created (#451). Sorry, guess I should have looked at the other active PR's.